tools/autograph-monitor/contentsignaturepki_test.go

```diff
@@ -136,12 +136,6 @@ var ValidMonitoringContentSignature = formats.SignatureResponse{
 	X5U:       "http://127.0.0.1:64320/normandychain",
 }
 
-// this is the trusted root ca for the firefox pki
-var FirefoxPKIRootPEM = []byte(firefoxPkiProdRoot)
```

Comment:
These are public, but I don't think anything else is using the monitor (e.g. it's not an importable module via go.mozilla.org)